test/models/user_test.rb

```diff
@@ -1,7 +1,32 @@
 require 'test_helper'
 
 class UserTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  
+  test "Name length" do 
+  	user = users(:bob)
+  	user.name = "j"*5
+  	assert_equal(-1 , (user.name).length <=> 50, "Your name must be less than 50 characters.")
```

Comment:
Instead of comparing the return code of `<=>` you can say

`assert user.name.length < 50, "Your name must be less than 50 characters"`